source/includes/APIReference/TimeOffRequests/_time_off_requests.md.erb

```diff
@@ -0,0 +1,19 @@
+# Time Off Requests
+
```

Comment:
Would you mind adding a summary description?  I know it's not part of your port, but it's really helpful for external users.  See "Files" or "Custom Fields" for examples.  We haven't been consistent with this in the past, but are hoping to improve moving forward as we release new endpoints such as this one.